lib/closure_tree/configuration.rb

```diff
@@ -0,0 +1,10 @@
+module ClosureTree
+  class Configuration # :nodoc:
+    attr_accessor :app_name, :database_less
+
+    def initialize
+      @app_name = 'Amazing app'
```

Comment:
In this momment that configuration is not used, first commit add the configuration file I thought would be weird a configuration file with no configurations that's why I added that also it can be deleted